packages/blade/src/components/Input/BaseInput/BaseInput.tsx

```diff
@@ -321,6 +351,61 @@ const autoCompleteSuggestionTypeValues = [
   'creditCardExpiryYear',
 ];
 
+type OnInputKeydownTagHandlerType = (key: string | undefined) => void;
+const useTags = (
+  tags: BaseInputProps['tags'],
+  activeTagIndex: number,
+  setActiveTagIndex?: (activeTagIndex: number) => void,
+): {
+  onInputKeydownTagHandler: OnInputKeydownTagHandlerType;
+  visibleTagsCountRef: React.MutableRefObject<number>;
+} => {
+  const visibleTagsCountRef = React.useRef<number>(0);
+
+  const onTagLeft = (): void => {
+    if (activeTagIndex < 0) {
+      setActiveTagIndex?.(visibleTagsCountRef.current - 1);
+    }
+
+    if (activeTagIndex > 0) {
+      setActiveTagIndex?.(activeTagIndex - 1);
+    }
+  };
+
+  const onTagRight = (): void => {
+    if (activeTagIndex < visibleTagsCountRef.current - 1) {
+      setActiveTagIndex?.(activeTagIndex + 1);
+    }
+  };
+
+  const onTagRemove = (): void => {
+    if (activeTagIndex >= 0 && activeTagIndex < visibleTagsCountRef.current && tags) {
+      tags[activeTagIndex].props.onDismiss({ tagIndex: activeTagIndex });
+    }
+  };
+
+  const onInputKeydownTagHandler: OnInputKeydownTagHandlerType = (key) => {
```

Comment:
Is all of this not causing any issue on RN? Check once?

Reply:
Nope. the keydown handler only exists for web so this is never getting triggered in RN